maxim/test_runs/run_utils.py

```
@@ -21,14 +21,14 @@ def get_variables_from_row(
         if column_type == "FILE_URL_VARIABLE":
             variables[column_name] = Variable(
-                type_="file", payload={"files": [{"url": row[column_name]}]}
+                type="file", payload={"files": [{"url": row[column_name]}]}
             )
         elif column_type in ("VARIABLE", "NULLABLE_VARIABLE"):
             # Skip nullable variables with None values to avoid invalid payloads
             if column_type == "NULLABLE_VARIABLE" and row[column_name] is None:
                 continue
             variables[column_name] = Variable(
-                type_="text", payload={"text": row[column_name]}
+                type="text", payload={"text": row[column_name]}
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)